src/utils.js

```diff
@@ -8,6 +8,10 @@
 import BigNumber from "bignumber.js";
 import { crypto } from "bitcoinjs-lib";
 
+const VALID_BASE64_REGEX = 
+  /^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{4})$/;
+const VALID_HEX_REGEX = /^[0-9A-Fa-f]*$/;
```

Comment:
Just want to comment that this could be performed in a single operation similarly to the base64 check (validating complete bytes) if the regex looked something like `/^([0-9A-Fa-f][0-9A-Fa-f])*$/`